### PR TITLE
New theme commands

### PR DESCRIPTION
**Pull request by andreytroeglazov**
#184
- Changed watching command to theme:watch - ddev theme:watch
- Added new command for compiling theme:compile - ddev theme:compile
- Updated readme

### CODE REVIEW

**Comment by andreytroeglazov** [comment]
Tested on jep-portal
Selection_517
Selection_516

File: .ddev/commands/web/theme:watch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 ## Description: Run Tailwind CSS compile, watch for changes.
-## Usage: tw
-## Example: "ddev tw"
+## Usage: theme:watch
```
Comment (amitaibu) [inline .ddev/commands/web/theme:watch:4]: As we will be watching only Tailwind, and not new images/fonts/JS. Maybe we should call `theme:watch:tw`? Or keep simple and leave as is?

Comment (andreytroeglazov) [reply .ddev/commands/web/theme:watch:4]: `theme:watch:tw` imo its a little bit complex and also we wont have `theme:watch`
I think its worth to update readme to let developers know that its watching only styles and leave it as `theme:watch`.

Comment (amitaibu) [reply .ddev/commands/web/theme:watch:4]: > I think its worth to update readme to let

Sounds good, go for it :)

Comment (andreytroeglazov) [reply .ddev/commands/web/theme:watch:4]: @amitaibu readme is updated now.

**Comment by amitaibu** [comment]
Thanks

**Pull request merged**